pkg/util/k8sutil/k8sutil.go

```diff
@@ -196,6 +196,9 @@ func NewMemberServiceManifest(etcdName, clusterName string, owner metatypes.Owne
 		ObjectMeta: v1.ObjectMeta{
 			Name: etcdName,
 			Labels: map[string]string{
+				// Member service is different from etcd (client) service.
+				// The owner of member service is the pod itself, so we can't use "app=etcd" which is used by GC.
+				"app":          "etcd-member-svc",
```

Comment:
> From in cluster view, each etcd member exposes two set of endpoints: peer/client. peer is for in cluster communication; client is for external communication with clients.

This is non-related to what's discussed.

Let's say I have 3 members: [00, 01, 02]. There are services:
```
etcd-cluster: [00, 01, 02]
etcd-cluster-00: [00]
etcd-cluster-01: [01]
etcd-cluster-02: [02]
```
Not related to peer/client communication.